targets/csharp/source/PlayFabSDK/source/PlayFabHttp/PlayFabPollyHttp.cs

```diff
@@ -92,12 +93,22 @@ public new async Task<object> DoPost(string fullPath, object request, Dictionary
         /// <summary>
         /// Overrides the Polly Policies to enforce.
         /// </summary>
-        /// <param name="retryPolicy">The retry policy to use.</param>
-        /// <param name="breakerPolicy">The circuit breaker policy to use.</param>
         /// <exception cref="ArgumentNullException"> Thrown when retryPolicy and/or breakerPolicy is null.</exception>
         public void OverridePolicies(AsyncPolicyWrap<object> policy)
         {
             CommonResilience = policy;
         }
+
+        private async Task<object> PollyDoPostAsync(string fullPath, object request, Dictionary<string, string> headers)
+        {
+            var result = await base.DoPost(fullPath, request, headers);
+            var resultAsError = result as PlayFabError;
+            if (resultAsError != null)
+            {
+                return resultAsError;
+            }
+
+            return result as PlayFabResultCommon;
```

Comment:
I still have concerns about this. I'm curious if this cast actually works in the success case. The success case test below may not have actually been invoking the polly plugin.

Reply:
https://learn.microsoft.com/en-us/rest/api/playfab/client/authentication/get-title-public-key?view=playfab-rest  looks like this api doesnt work anymore. Its the only api that requires no auth, which was what i intended to get. 

Found another endpoint i can use.

Reply:
Updated to check for success case.